src/lib/y2storage/align_policy.rb

```diff
@@ -7,6 +7,6 @@ module Y2Storage
   class AlignPolicy
     include StorageEnumWrapper
 
-    wrap_enum "Storage::AlignPolicy", names: [:align_end, :keep_end, :keep_size]
+    wrap_enum "AlignPolicy"
```

Comment:
If you mean turning `StorageEnumWrapper` from a mixin into a class, it makes sense if you look at it in isolation. But I like the fact that the implementation is symmetric to `StorageClassWrapper` (both implemented as modules).

If you mean adding a base class in addition to the mixin (so basically a one-liner class), we could do it, but I'm not sure if it's worth it.